tests/test.py

```diff
@@ -0,0 +1,38 @@
+import time
+import logging
+import os
+import subprocess
+
+from avocado import VERSION
+from avocado import Test
+
+class MinishiftISOTest(Test):
+
+    def setUp(self):
+        ''' Test Setup '''
+        self.test_DIR = os.path.dirname(os.path.realpath(__file__))
+        self.log.info("################################################################")
+        self.log.info("Avocado version : %s" % VERSION)
+        self.log.info("################################################################")
+
+    def test_minishift_centos_iso_build(self):
+        ''' minishift centos iso build test '''
+        self.log.info("Testing minishift centos iso build ...")
+        process = subprocess.Popen("make", shell=True, stdout=subprocess.PIPE)
```

Comment:
@hferentschik looks like the target is `make iso_creation` and it is failing currently.

@budhrg it is a normal practice in Makefiles to keep the build (which is ISO creation in our case) and tests separate. Also `make test` should trigger the tests you have written.